### ISSUE TITLE
Include reproduction information

### ISSUE THREAD

**Issue by agnosticlines** [pizlonator/pizlonated-zsh#1]
Hey! I've been reading for pizlonated forks of projects to try understand how to port software to it, it's quite a pain trying to figure out exactly which version/commit some of the software is, would it be possible to include a git hash so someone can just get a .patch file to see all the changes?

Thanks

**Comment by pizlonator** [comment]
You can figure out where each port is forked from by doing a git log. The first commit you find that isn't by me is where it forked from, and then you can run a diff to see everything that changed.

**Comment by agnosticlines** [comment]
> You can figure out where each port is forked from by doing a git log. The first commit you find that isn't by me is where it forked from, and then you can run a diff to see everything that changed.

Yep for zsh that works, but for some of the other projects such as curl it doesn't as you've not included the commit history https://github.com/pizlonator/deluded-curl-8.5.0

**Comment by pizlonator** [comment]
Then the version I branched from was 8.5.0 and the first commit was without any Fil-C changes.
